### ISSUE TITLE
Accessible tooltips

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/yari#2705]
This is a follow up from https://github.com/mdn/content/issues/987#issuecomment-768313842

So far, I've noticed the usage of `title` attribute in some places on MDN web docs.
Those are not accessible for non-hover devices or screenreaders.

However, its usage hint that there is some value in it.
Thus, I'd like us to explore the use cases and constraints of those tooltips.

An accessible and inclusive markup is suggested in https://inclusive-components.design/tooltips-toggletips/ which could be use as a starting point.

I'm not sure, whether an implementation would end up in Yari, Content or Minimalist. (Likely a combination of all of them - design in minimalist, hints in Content which then would be transformed by Yari).

**Comment by schalkneethling** [comment]
Thank you for opening this issue @Ryuno-Ki. This is something I have been thinking about and I do have some ideas. I will follow up with some ideas.

**Comment by Ryuno-Ki** [comment]
Looking forward to it :)

**Comment by mattiapontonio** [comment]
Challenge ⚔️: CSS only.
